### ISSUE TITLE
[Twig/XML] Twig 3.x breaks our XML scripting

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#1168]
Test case:

~~~
{% set string_of_xml =
"<response xmlns=\"http://www.example.com/api/\">
  <client_id>1</client_id>
  <invoice_id>123</invoice_id>
</response>"
-%}
{% set xml = xml_decode(string_of_xml) %}
{{xml_encode(xml.client_id)}}
~~~

**Comment by jstanden** [comment]
We need to implement `SecurityPolicyInterface` rather than override final `SecurityPolicy`.

**Comment by jstanden** [comment]
Fixed in 9.5 with our own security policy.
